Finish round 1 of Darwin Core mapping

peterdesmet opened the pull request:
Things done:
1. Create 3 files for mapping: `change-fields`, `add-fields` and `delete-fields` (commit 98bbb7b)
2. Move all fields from `obis-to-dwc-mapping` to one of these 3 files and remove original file (done in commit 9c5a906)
3. Review **all terms** in `dwc-mapping` and indicate status with `-` (don't use) or `suggested` (suggest to use). (finished in commit 9268a6e)

This finishes our round 1 of Darwin Core mapping. Now we need feedback from the Diveboard team, to see which suggested terms can actually be used, and which ones cannot.

@DimEvil, can you review my commits, in particular the ones with `Don't use` which often have extra explanation why I recommend **not** to use a term? If all looks good, you can merge this request.

peterdesmet committed:
Split mapping in 3 files: add, change, delete

25 fields moved

peterdesmet committed:
13 fields split over delete and change

peterdesmet committed:
32 more fields moved to add, change and delete

peterdesmet committed:
Replace n/a with -

For readibility

peterdesmet committed:
Delete subgenus

This term is seldom used and is of little use

peterdesmet committed:
Update status of DwC terms

peterdesmet committed:
Add applicable resource terms to add-fields

peterdesmet committed:
Don't use accessRights

peterdesmet committed:
Don't use informationWithheld, dataGeneralizations

peterdesmet committed:
Don't use dynamicProperties

peterdesmet committed:
Correct typo

peterdesmet committed:
Don't use higherGeography

Little value

peterdesmet committed:
Don't use island & islandGroup

Will be difficult to obtain from country, location, spot name and
sea/lake

peterdesmet committed:
Add countryCode

peterdesmet committed:
Add locationAccordingTo

peterdesmet committed:
Add geodeticDatum

peterdesmet committed:
Don't use georeference Terms

Will be hard to obtain (GPS, added manually, etc.)

peterdesmet committed:
locality = divespot + location

peterdesmet committed:
Include elevationInMeters

I noticed altitude information can be provided in meters, so it makes
sense to include these fields

peterdesmet committed:
Correct format issue

peterdesmet committed:
Add remark regarding surface/bottom temparature

See also issue #6

peterdesmet committed:
Add remark regarding duration

I noticed time is noted as startTime + duration.

peterdesmet committed:
Don't use identification properties

It is unlikely more information can be provided than identifiedBy and
dateIdentified.

peterdesmet committed:
Use taxonID, don't use taxonConceptID

Relates to the taxon class as a whole. Since we get all info from EOL,
it is best to use this term and not `taxonConceptID

peterdesmet committed:
Correct typo

peterdesmet committed:
Use taxonID

peterdesmet committed:
Use nameAccordingToID

peterdesmet committed:
Use vernacularName

peterdesmet committed:
Don't use nomenclaturalCode

Unknown and across several kingdoms

peterdesmet committed:
Add taxonRank

peterdesmet committed:
Suggest behavior

peterdesmet committed:
Map recordID to catalogNumber, not occurrenceID

occurrenceID should be used to, but globally unique

peterdesmet committed:
Use occurrenceID

peterdesmet committed:
Use associatedMedia

peterdesmet committed:
Use eventID

peterdesmet committed:
Remark about occurrenceID

peterdesmet committed:
Use fieldNotes

peterdesmet committed:
Don't use habitat

peterdesmet committed:
Don't use samplingProtocol

We could use `ad hoc observation` as a fixed value, but we cannot
guarantee all occurrences are sampled that way. Suggest we do not use
this @DimEvil.

peterdesmet commented:
PS: I also created some issues.

DimEvil commented on standardization/add-fields.md:
```diff
@@ -0,0 +1,32 @@
+n | ownerInstitutionCode | Diveboard | 
+n | countryCode | ES | Depends on how country data are stored
+n | locationAccordingTo | Wikipedia | Depends on how location data are stored
+n | geodeticDatum | EPSG:4326 | `fixed value` or depending on GPS
```
The coördinates are generated through google maps by clicking. These are typically the coordinates of the diversite. All the coördinates of one dive will be the same. therefore a uncertainty of 100 meters is reasonable...  (as a divesite is usually not bigger 1/4 km²)

peterdesmet commented on standardization/add-fields.md:
```diff
@@ -0,0 +1,32 @@
+n | continent | Europe | 
+n | waterBody | North Sea | 
+n | locality | Cavall Bernat, Estartit & Medes Islands | Name of divespot, location
+n | occurrenceRemarks | One fin missing | Remarks about the observed species
```
Remarks cannot be added on the level of observed species.

DimEvil commented on standardization/add-fields.md:
```diff
@@ -0,0 +1,32 @@
+n | waterBody | North Sea | 
+n | locality | Cavall Bernat, Estartit & Medes Islands | Name of divespot, location
+n | occurrenceRemarks | One fin missing | Remarks about the observed species
+n | behavior | Remarks about the behavior of the observed species
```
Remarks cannot be added on the level of observed species.

peterdesmet committed:
Don't use occurrenceRemarks

As
[commented](https://github.com/Datafable/diveboard-gbif/pull/8/files#r55
76658), no information can be added by the user on the level of
observed species.

peterdesmet committed:
Don't use behavior

As commented, no information can be added by the user about an observed
species

peterdesmet committed:
Suggest habitat

As
[commented](https://github.com/Datafable/diveboard-gbif/commit/d9730b456
20b7b2269f5f4f360a97db631b01fcb#commitcomment-3786985), habitat might
be suggested, with a controlled vocabulary.

DimEvil commented on standardization/add-fields.md:
```diff
@@ -0,0 +1,31 @@
+n | locationAccordingTo | Wikipedia | Depends on how location data are stored
+n | geodeticDatum | EPSG:4326 | `fixed value` or depending on GPS
+n | minimumElevationInMeters | 313 | 
+n | maximumElevationInMeters | 313 | Identical to `minimumElevationInMeters`
```
I would certainly drop these elements.

peterdesmet replied:
Why? This is already available in the interface, so the information can be provided by the users.

DimEvil replied:
you mean, the altitude field. I didn t see that. 
Most of the dives are done in seawater, this means elevation zero. 
But, ok, i agree, we can leave it in, we might need another column in this file (original Diveboardfield orso..). T

DimEvil commented on standardization/add-fields.md:
```diff
@@ -0,0 +1,31 @@
+n | taxonRank | If provided by EOL
+n | vernacularName | If provided by EOL
+n | occurrenceID | Diveboard:20745099 | 
+n | associatedMedia | `image-url; image-url` | To be discussed:  should refer to the occurrence, not event
```
This can refer to pictures hosted on "Diveboard" uploaded by divers. This moment, pictures are linked with the event, not the occurrence.... Might be complicated to accomplish

peterdesmet replied:
Ok, I'll leave it in for now. To be discussed with Diveboard.

DimEvil committed:
Update change-fields.md

Changed OBIS field to DIVEBOARD field

DimEvil commented on standardization/change-fields.md:
```diff
@@ -0,0 +1,41 @@
+n | ContinentOcean | higherGeographyID | #<Region:0x00000006498760> | `idem` | To what does this refer exactly?
+n | Country | country | Egypt | `idem` | 
+n | Locality | locationID | #<Location:0x000000068d9c98> | `idem` | To what does this refer exactly?
+n | Longitude | decimalLongitude | 34.513 | `idem` | Are these numbers rounded?
```
I think they are rounded, we should decide on what comes after the ","
I would suggest 5 digits and 100 m uncertainty

DimEvil commented on standardization/change-fields.md:
```diff
@@ -0,0 +1,41 @@
+n | Longitude | decimalLongitude | 34.513 | `idem` | Are these numbers rounded?
+n | Latitude | decimalLongitude | 28.501 | `idem` | Are these numbers rounded?
+n | CoordinatePrecision | coordinateUncertaintyInMeters | 100 | `idem` | Verify these are meters
+n | MinimumDepth | minimumDepthInMeters | 0 | `idem` | Verify these are meters
```
no worries, it s meter peter :D 
minimumDepthInMeter might be obsolete, I would drop that line.

peterdesmet replied:
:tongue: It's suggested to populate both fields, especially if we use maximumDepthInMeters.

DimEvil replied:
ff direct reply via de mail proberen!

I think you mean: use MinimumDepthinmeters because we use
MaximumDepthinMeters. I think this "field" is typically for marine datasets
were they take samples in a specific depth range.
Another thing which just crossed my mind; I think that we also must specify
in every record that it is citizen science. can we place that somewhere?
Maybe use occurrenceRemarks (it looks like that one is disappeared :) )

Chrs,
Dimi

and btw: who is suggesting that?

On Mon, Aug 5, 2013 at 5:01 PM, Peter Desmet notifications@github.comwrote:

> In standardization/change-fields.md:
> 
> > +n | Subspecies | infraspecificEpithet | | `?` | If provided by EOL
> > +n | ScientificNameAuthor | scientificNameAuthorship | | `?` | If provided by EOL
> > +n | IdentifiedBy | identifiedBy | | Bruno Lestrade | Identical to `recordedBy`
> > +n | Collector | recordedBy | Bruno Lestrade - Diveboard | Bruno Lestrade |
> > +n | YearCollected | eventDate | 2005 | 2005-07-25 | Merge data into `eventDate`
> > +n | MonthCollected | eventDate | 7 | 2005-07-25 | Merge data into `eventDate`
> > +n | DayCollected | eventDate | 25 | 2005-07-25 | Merge data into `eventDate`
> > +n | StartTimeOfDay | eventTime | 0 | 14:03:02Z | Merge data into `eventTime`
> > +n | EndTimeOfDay | eventTime | 29/60 | 14:03:02Z | Merge data into `eventTime` with `/` if duration is provided
> > +n | ContinentOcean | higherGeographyID | #Region:0x00000006498760 | `idem` | To what does this refer exactly?
> > +n | Country | country | Egypt | `idem` |
> > +n | Locality | locationID | #Location:0x000000068d9c98 | `idem` | To what does this refer exactly?
> > +n | Longitude | decimalLongitude | 34.513 | `idem` | Are these numbers rounded?
> > +n | Latitude | decimalLongitude | 28.501 | `idem` | Are these numbers rounded?
> > +n | CoordinatePrecision | coordinateUncertaintyInMeters | 100 | `idem` | Verify these are meters
> > +n | MinimumDepth | minimumDepthInMeters | 0 | `idem` | Verify these are meters
> 
> [image: :tongue:] It's suggested to populate both fields, especially if
> we use maximumDepthInMeters.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/Datafable/diveboard-gbif/pull/8/files#r5580677
> .

DimEvil commented on standardization/change-fields.md:
```diff
@@ -0,0 +1,41 @@
+n | MinimumDepth | minimumDepthInMeters | 0 | `idem` | Verify these are meters
+n | MaximumDepth | maximumDepthInMeters | 12.0 | `idem` | Verify these are meters
+n | Temperature | - | 28.0 | `idem` | This can be provided in a `MeasurementOrFacts` extension, requires some work (this might be split in surface and bottom temp.)
+n | Sex | sex | | female | Currently not recorded, to be discussed
```
as long as we add species in bulk, we can not comment on each species. No Sex this time.

DimEvil commented on standardization/change-fields.md:
```diff
@@ -0,0 +1,41 @@
+n | MaximumDepth | maximumDepthInMeters | 12.0 | `idem` | Verify these are meters
+n | Temperature | - | 28.0 | `idem` | This can be provided in a `MeasurementOrFacts` extension, requires some work (this might be split in surface and bottom temp.)
+n | Sex | sex | | female | Currently not recorded, to be discussed
+n | LifeStage | lifeStage | | adult | Currently not recorded, to be discussed
```
as long as we add species in bulk, we can not comment on each species.
We might suggest to diveboard; the possibility to comment on each species,

peterdesmet committed:
Don't use sex

Occurrence level info cannot be provided.

peterdesmet committed:
Don't use lifeStage

Occurrence level information cannot be provided.

peterdesmet committed:
Forgot to add sex/LifeStage to delete-fields

peterdesmet committed:
Don't use individualCount

Users cannot add occurrence information

peterdesmet committed:
geodeticDatum = EPSG:3857

All coordinates are added by clicking on Google Maps, which has datum
[EPSG:3857](http://wiki.openstreetmap.org/wiki/EPSG:3857)

peterdesmet committed:
Suggest georeferenceSources with fixed value

peterdesmet committed:
Add comment on coordinateUncertaintyInMeters

See also [this
comment](https://github.com/Datafable/diveboard-gbif/pull/8#discussion_r
5576587) by @DimEvil
